pepper-apis/src/main/java/org/broadinstitute/ddp/model/user/UserProfile.java

```diff
@@ -0,0 +1,152 @@
+                       @ColumnName("last_name") String lastName,
+                       @ColumnName("sex") SexType sexType,
+                       @ColumnName("birth_date") LocalDate birthDate,
+                       @ColumnName("preferred_language_id") Long preferredLangId,
+                       @ColumnName("iso_language_code") String preferredLangCode,
+                       @ColumnName("do_not_contact") Boolean doNotContact,
+                       @ColumnName("is_deceased") Boolean isDeceased) {
+        this.userId = userId;
+        this.firstName = firstName;
+        this.lastName = lastName;
+        this.sexType = sexType;
+        this.birthDate = birthDate;
+        this.preferredLangId = preferredLangId;
+        this.preferredLangCode = preferredLangCode;
+        this.doNotContact = doNotContact;
+        this.isDeceased = isDeceased;
+    }
+
+    public long getUserId() {
+        return userId;
+    }
+
+    public String getFirstName() {
+        return firstName;
+    }
+
+    public String getLastName() {
+        return lastName;
+    }
+
+    public SexType getSexType() {
+        return sexType;
+    }
+
+    public LocalDate getBirthDate() {
+        return birthDate;
+    }
+
+    public Long getPreferredLangId() {
+        return preferredLangId;
+    }
+
+    public String getPreferredLangCode() {
+        return preferredLangCode;
+    }
+
+    public Boolean getDoNotContact() {
+        return doNotContact;
+    }
+
+    public Boolean getIsDeceased() {
+        return isDeceased;
+    }
+
+    public enum SexType {
+        FEMALE, MALE, INTERSEX, PREFER_NOT_TO_ANSWER
+    }
+
+    public static class Builder {
+        private UserProfile profile;
+
+        public Builder(long userId) {
+            profile = new UserProfile(userId, null, null, null, null, null, null, null, null);
+        }
+
+        public Builder(UserProfile other) {
+            profile = new UserProfile(
+                    other.getUserId(),
+                    other.getFirstName(),
+                    other.getLastName(),
+                    other.getSexType(),
+                    other.getBirthDate(),
+                    other.getPreferredLangId(),
+                    other.getPreferredLangCode(),
+                    other.getDoNotContact(),
+                    other.getIsDeceased());
+        }
+
+        public Builder setFirstName(String firstName) {
+            profile.firstName = firstName;
+            return this;
+        }
+
+        public Builder setLastName(String lastName) {
+            profile.lastName = lastName;
+            return this;
+        }
+
+        public Builder setSexType(SexType sexType) {
+            profile.sexType = sexType;
+            return this;
+        }
+
+        public Builder setBirthDate(LocalDate birthDate) {
+            profile.birthDate = birthDate;
+            return this;
+        }
+
+        public Builder setPreferredLangId(Long preferredLangId) {
```

Comment:
can't we just remove the set method if we are not using the value passed and leave it initialized to null to avoid unnecessary confusion !!